### DISCUSSION TITLE
Streaming: Feature Availability

### DISCUSSION THREAD

**Discussion by lilyydu** [microsoft/teams-sdk#2069] [Announcements]
Hi everyone,

We are excited to announce that we have recently published some streaming logic in all 3 languages. Currently, this feature is in the rollout phase and is available only to specific tenants. Our team is actively working on enabling this feature fully on Teams.

AI SDK aims to empower partners with access to streaming features. However, please be aware that this may lead to a broken experience for those who do not yet have access. Rest assured; we are diligently working to enable this feature for everyone.

Once the feature is generally available, we will make a public announcement. We apologize for any confusion and appreciate your patience.

**Comment by heyitsaamir** [comment]
We're clearing discussions and starting fresh. This discussion has not been acted upon. Please reopen the discussion, or start a new one if this question is still important. Thank you for your understanding.
